Week3/homework/step2-2.js

```diff
@@ -3,18 +3,27 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
 
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let number = startIndex; number <= stopIndex; number++) {
+    numbers.push(number); // This create an array [10,11,12,13,14,15]
+  }
+  // Set the conditions of the callback for threeCallback, fiveCallback.
+  for (let i = 0; i < numbers.length; i++) {
+    const number = numbers[i];
+    if (number % 3 === 0) {
+      threeCallback(number);
+    }
+    if (number % 5 === 0) {
+      fiveCallback(number);
+    }
+  }
```

Comment:
Ah. Couple things here:
1. You don't need to have two separate loops for adding numbers to the array and then running the `if-else` condition. You could do both operations in a single loop.
2. You missed the requirement of printing when a number is divisible by both 3 and 5. Could you please add the same? **Hint**: a number divisible by both 3 and 5 will be divisible by 15 too. 😉

Reply:
Thanks for the hints. :)